### ISSUE TITLE
translate-c fails when including variables with the name `undefined`

### ISSUE THREAD

**Issue by mparadinha** [ziglang/zig#11792]
### Zig Version

0.10.0-dev.2473+e498fb155

### Steps to Reproduce

in a file `struct.h`:
```
typedef struct {
    int undefined;
} WithUndef;
```
(also fails if we use `true` or `false` instead of `undefined`)

main.zig:
```
const c = @cImport({
    @cInclude("struct.h");
});

pub fn main() void {
    const s: c.WithUndef = undefined;
    _ = s;
}
```

### Expected Behavior

Successful compilation

### Actual Behavior

```
$ zig build-exe main.zig -I.
zig-cache/o/d7e43d10586c29c4899d69d9dafcc319/cimport.zig:56:5: error: expected token '}', found 'undefined'
    undefined: c_int,
    ^
```

**Comment by r00ster91** [comment]
The Zig output should use `@""`.

**Comment by mparadinha** [comment]
This behavior is fixed in 0.10
